hs628_tools/Visualization.py

```diff
@@ -0,0 +1,294 @@
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+import pandas as pd
+import sys 
+import plotly.plotly as py
+import plotly.graph_objs as go
+#plotly.tools.set_credentials_file(username='hhowell', api_key='BRP1aHhfTHwkqnvalVQL')
+from sklearn import datasets, linear_model
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import mean_squared_error,mean_absolute_error, r2_score
+###This is importing a class from Issue #11 Test module. We used the ratio function to determine the test size ratio for the train_test_split function. 
+#from Test import data_split as ds
+
+
+###Bar Plot-Count
+#This function counts all instances in a variable and creates a barplot to display the results 
+#@param: The variable to be counted, the x_axis label, title of the graph
+#@return: A count bar plot of the variable inputted 
+def count_and_print(variable, x_label, title):
+    def count(variable):
+        new_dic = {}
+        c = (len(variable)-1)
+        for index, value in enumerate(variable):
+            count = 0
+            if((value in new_dic) == True):
```

Comment:
I think someone else already noted the ```pass``` here was slightly awkward and could be avoided by inverting the logic of your test or changing the block structure.  There is also no need for the ```== True``` 

In a loop like this, with no code following the if block, it can often be slightly cleaner code to replace 
```python
    if value not in new_dic:
        for index2, value2 in enumerate(variable):
        ...
```
with 
```python
    if value in new_dic:
        continue
    for index2, value2 in enumerate(variable):
```